runtime/src/main/java/com/datastax/oss/driver/shaded/guava/common/primitives/UnsignedBytesSubstitutions.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.driver.shaded.guava.common.primitives;
+
+import com.oracle.svm.core.annotate.Alias;
+import com.oracle.svm.core.annotate.Delete;
+import com.oracle.svm.core.annotate.RecomputeFieldValue;
+import com.oracle.svm.core.annotate.Substitute;
+import com.oracle.svm.core.annotate.TargetClass;
+import java.util.Comparator;
+
+@TargetClass(UnsignedBytes.LexicographicalComparatorHolder.UnsafeComparator.class)
+@Delete
+final class UnsafeComparatorSubstitutions {}
```

Comment:
One _could_ argue that marking this class as deleted (or more accurately "unavailable once this substitution is applied") is a bit heavy-handed.  I think it's a nice bit of insurance; with this step any code path which _could_ lead to an instance of this class will trigger build failures in Graal's analysis phase.  This is desirable; it _forces_ the more rigorous impl below.

Reply:
I'm fine with the principle, but shouldn't this substitution go into the driver itself?

Reply:
Hmmm, that's an _interesting_ point!  Doing so would benefit any other potential users of the Guava classes as well... I kinda like that.

Perhaps we do it here for now (to get the build fixed) _and_ in the driver and then remove this code when we get a version of the driver that contains the same thing?

Reply:
I've confirmed that adding the substitution to the driver core will work.  I created a local 4.7.0-SNAPSHOT instance with the substitution in place and removed it elsewhere... and the native build was still clean.  I created a PR to represent what was done to the core to make that work... that's available at https://github.com/datastax/java-driver/pull/1443

Note a few things about that change:

* We have to add a new dependency to the core driver, although I'm pretty sure we should be able to add it in "provided" scope (still need to test that)
* I encountered a weird Javadoc generation error which required excluding the new substitution from the javadoc plugin.  I don't think this is a problem as this seems like an internal impl detail and not something we'd want to document necessarily... but it did strike me as strange.  More details on the PR referenced above.
* Adding the substitution to the core now splits the Guava package across two JARs which makes Felix and the bundler plugin unhappy... again, more details on the PR.

I think this idea is still quite viable, however I did want to be very explicit about the above behaviours to make sure we were okay with them.